migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs

```diff
@@ -417,7 +417,13 @@ impl SqlRenderer for PostgresFlavour {
 
 fn render_column_type(col: &ColumnWalker<'_>, flavour: &PostgresFlavour) -> Cow<'static, str> {
     let t = col.column_type();
-    let is_autoincrement = col.is_autoincrement();
+
+    let is_unique_rowid = col
+        .default()
+        .map(|d| matches!(d.kind(), DefaultKind::DbGenerated(s) if s == "unique_rowid()"))
+        .unwrap_or(false);
+
+    let is_autoincrement = col.is_autoincrement() && !(is_unique_rowid && flavour.is_cockroachdb());
```

Comment:
The `is_autoincrement()` will translate that to a `SERIAL`, right? If so, it's wrong, because it doesn't create the type of primary key that is expected (sequence vs rowid).